### PR TITLE
Bug 1621062 - Bump version to 9.0.0

### PR DESCRIPTION
**Pull request by miketaylr**
r? @denschub 

(my prettier-vscode installation insisted on formatting the JSON file, seems ok?)

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -89,4 +128,4 @@
       "run.js"
     ]
   }
-}
+}
```
Comment (denschub) [inline src/manifest.json]: Did prettier remove the newline at the end? That's... odd.

Comment (miketaylr) [reply src/manifest.json]: (my VS-code might have some funky settings....)

**Comment by miketaylr** [comment]
Hm, why is abde5da in this PR????

**Comment by denschub** [comment]
r+ for the one commit updating the manifest. I'll do the merge and cherry-pick only the right commit, so you don't have to fight with git. One second. :)

**Comment by miketaylr** [comment]
thanks @denschub!

**Pull request merged**